### PR TITLE
docs: enable theme selection on the doc site

### PR DESCRIPTION
**Pull request by droshev**
enable theme selection on the doc site

### CODE REVIEW

**Comment by mikerodonnell89** [comment]
Yea that is a strange place for that.  I think maybe a gear icon in the top right where the github logo is

**Comment by estelaxu3** [comment]
Two things I noticed on this page: 
1. The Github version text color should change accordingly in High Contrast White
2. The browser tab icon's background changes when switching the theme, I don't think it is necessary since the logo represents entire library including Styles. If we really want that, maybe we should switch back to white back ground in High Contrast White theme so it is consistent. 
Screen Shot 2020-05-12 at 10 09 53 AM

File: apps/docs/src/app/documentation/core-helpers/sections-toolbar/sections-toolbar.component.ts [hunk 1]
```diff
@@ -52,4 +68,9 @@ export class SectionsToolbarComponent implements OnInit {
     onResize() {
         this.windowSize();
     }
+
+    selectTheme() {
+        const cssLink: any = document.getElementById('themes');
+        cssLink.href = 'assets/' + this.selectedTheme + '.css';
```
Comment (rengare) [inline apps/docs/src/app/documentation/core-helpers/sections-toolbar/sections-toolbar.component.ts:74]: > cssLink.href = `assets/${this.selectedTheme}.css`

**Comment by salarenko** [comment]
blinking-theme-select

**Comment by estelaxu3** [comment]
The popover is has white top and button space. 
Screen Shot 2020-05-14 at 8 56 44 AM

**Comment by estelaxu3** [comment]
looks good to me.

**Comment by InnaAtanasova** [comment]
Screen Shot 2020-05-14 at 5 38 50 PM

Shouldn't this button be like the other buttons in the Shellbar? (the hover/active state)

**Pull request merged**